### ISSUE TITLE
no option to ignore suspend certain tabs

### ISSUE THREAD

**Issue by 4evermaat** [piroor/suspendtab#56]
not based only on domain name.  We need ability to ignore tabs:
1) if they are in focus.  So the currently focused tab is exempt from suspension
2) If any text fields have been filled in. If the user is filling in forms, tab is exempt from suspension.

**Comment by ssokolow** [comment]
It'd also be nice to have a "Ignore suspension of this (playing) YouTube video but still suspend all of the other ones I'm using tabs to queue up"-style behaviour.

**Comment by piroor** [comment]
@4evermaat I agree that both cases shouldn't be suspended. I designed this to keep the focused (current) tab loaded but accidentally unloaded... I have to fix it.

@ssokolow I agree this also. I think "playing" tabs should be on the memory always.
